highway_env/envs/common/graphics.py

```diff
@@ -109,7 +122,7 @@ def get_image(self):
         """
         :return: the rendered image as a rbg array
         """
-        data = pygame.surfarray.array3d(self.screen)
+        data = pygame.surfarray.array3d(self.sim_surface)
```

Comment:
`array3d(self.sim_surface if self.offscreeen else self.screen)`
Usually the screen surface also contains an agent_surface that allows to render the agent decisions at test_time.